src/Templates/Boilerplate/Bit.Boilerplate/.devcontainer/devcontainer.json

```
@@ -1,6 +1,10 @@
 {
     "name": "Boilerplate",
+    //#if (framework == 'net9.0')
+    "image": "mcr.microsoft.com/dotnet/sdk:9.0",
```

Biome

[error] The key image was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)